client/extensions/wp-super-cache/state/settings/reducer.js

```diff
@@ -1,12 +1,8 @@
 /**
- * External dependencies
- */
-import { combineReducers } from 'redux';
-
-/**
  * Internal dependencies
  */
 import { createReducer } from 'state/utils';
+import { combineReducersWithPersistence } from 'state/utils';
```

Comment:
or maybe update the codemod to automatically find imports from `state/utils` and join into the list alphabetically